lib/taskana-core/src/test/java/acceptance/task/UpdateTaskAccTest.java

```diff
@@ -129,6 +130,31 @@ void should_ThrowException_When_TaskHasAlreadyBeenUpdated()
         .withFailMessage("The task has already been updated by another user");
   }
 
+  @WithAccessId(user = "admin")
+  @WithAccessId(user = "taskadmin")
+  @TestTemplate
+  void should_UpdateTask_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
+      throws TaskNotFoundException, ClassificationNotFoundException, InvalidArgumentException,
+                 ConcurrencyException, NotAuthorizedException, AttachmentPersistenceException,
+                 InvalidStateException, SQLException {
+
+    TaskService taskService = taskanaEngine.getTaskService();
+    Task task = taskService.getTask("TKI:000000000000000000000000000000000000");
+    final ClassificationSummary classificationSummary = task.getClassificationSummary();
+    task.setClassificationKey("T2100");
+    Task updatedTask = taskService.updateTask(task);
+    updatedTask = taskService.getTask(updatedTask.getId());
+
+    assertThat(updatedTask).isNotNull();
+    assertThat(updatedTask.getClassificationSummary().getKey()).isEqualTo("T2100");
+    assertThat(updatedTask.getClassificationSummary()).isNotEqualTo(classificationSummary);
+    assertThat(updatedTask.getCreated()).isNotEqualTo(updatedTask.getModified());
+    assertThat(task.getPlanned()).isEqualTo(updatedTask.getPlanned());
+    assertThat(task.getName()).isEqualTo(updatedTask.getName());
+    assertThat(task.getDescription()).isEqualTo(updatedTask.getDescription());
+    resetDb(false); // classification of task TKI:0..00 was changed...
```

Comment:
this should not be necessary if each test works own dedicated data sets.

Reply:
Was able to remove this without repurcussions - looks like it was indeed unnecessary